scripts/viz/feature_activation_plots.R

```diff
@@ -83,22 +96,23 @@ met_df$label[(met_df$acronym != "SKCM") &
                (met_df$sample_type == "Metastatic")] <- "Metastatic Other"
 met_df$label[(met_df$label == "MetastaticSKCM")] <- "Metastatic SKCM"
 
-p <- ggplot(met_df, aes(x = met_df$`53`, y = met_df$`66`, color = label)) + 
+p <- ggplot(met_df, aes(x = `53`, y = `66`, color = label)) + 
   geom_point() +
   xlab(paste("encoding 53")) +
   ylab(paste("encoding 66")) +
   theme_bw() +
   theme(text = element_text(size = 20))
 
 # Save plots with and without legend
-p + theme(legend.position = "none")
-ggsave(sample_type_file, width = 6, height = 5)
+p_legend <- p + theme(legend.position = "none")
+ggsave(sample_type_file, plot = p_legend, width = 6, height = 5)
 
-p
-ggsave(sample_legend_file, width = 6, height = 5)
+ggsave(sample_legend_file, plot = p, width = 6, height = 5)
 
 # 2) Full heatmap of samples by tybalt feature activation
-mat <- as.matrix(combined_df[, 2:101])
+encodings_df <- combined_df %>% select(num_range('', 1:100))
```

Comment:
For consistency with the rest of this script: `dplyr::select`

Reply:
oh yes, duh!